roles/user_config/README.md

```diff
@@ -0,0 +1,40 @@
+## Ansible Role: SASL Users and ACLs Manager
+This Ansible role is designed to manage Redpanda Cluster SASL users and ACLs. It provides the capability to create and delete users as well as their permissions.
+
+## Requirements
+Ansible 2.x
+
+## Security Considerations
+
+### Admin Credentials
+Ensure that the sasl_admin_username and sasl_admin_password are secure and are not hardcoded in the playbook
+
+### Password Management: 
+Ensure that passwords are managed securely. You can use a base64 encoded JSON object passed through the CLI to ensure secure password passthrough.
+
+## Role Variables
+
+| Variable                       | Description                                                                                   | Default                |
+|--------------------------------|-----------------------------------------------------------------------------------------------|------------------------|
+| `sasl_mechanism`               | The SASL mechanism to be used.                                                                | `SCRAM-SHA-256`        |
+| `acl_user_create`              | A switch to toggle between creating or deleting.                                              | `true`                 |
```

Comment:
It seems a bit odd that this being false would enable deletion -- I'd expect that this being false only opts out of creating, but does not additionally opt into deleting -- and I'm not sure of the difference between acl_user_create and delete_users in this readme

Reply:
The intent here is that you have to both disable user creation and enable user deletion to delete users. So there's two flags that need to be changed as a failsafe against accidental deletion. Setting acl_user_create to false alone isn't enough to enable deletion on its own.

Reply:
this does seem a bit odd. if we look at other ansible modules out there doing user management, we'd only need to key off state=present vs state=absent to create or delete. what am I missing here?